src/app/components/authorization-page/authorization-page.component.ts

```diff
@@ -42,4 +58,20 @@ export class AuthorizationPageComponent {
     return null;
   }
 
+  private _buildForm() {
+    if (this.isRegistration) {
+      this.form = this.formBuilder.group({
+        username: ['', [Validators.required]],
```

Comment:
@KovalivNataliia add pattern validator for username field. It must consist only characters of english alphabet.